picasso/src/test/java/com/squareup/picasso/BitmapHunterTest.java

```diff
@@ -873,6 +873,28 @@
     assertThat(shadowBitmap.getCreatedFromBitmap()).isNotSameAs(source);
   }
 
+  @Test public void onlyScaleDownResizeWidthIs0() {
+    Bitmap source = Bitmap.createBitmap(50, 50, ARGB_8888);
+    Request data = new Request.Builder(URI_1).resize(0, 60).onlyScaleDown().build();
+    Bitmap result = transformResult(data, source, 0);
+    assertThat(result).isSameAs(source);
+
+    ShadowBitmap shadowBitmap = shadowOf(result);
+    assertThat(shadowBitmap.getCreatedFromBitmap()).isNull();
+    assertThat(shadowBitmap.getCreatedFromBitmap()).isNotSameAs(source);
+  }
+
+  @Test public void onlyScaleDownResizeHeightIs0() {
+    Bitmap source = Bitmap.createBitmap(50, 50, ARGB_8888);
+    Request data = new Request.Builder(URI_1).resize(60, 0).onlyScaleDown().build();
```

Comment:
Can you also add two tests where it actually still needs scaled? Like for `40, 0` and `0, 40`?